spec/feature/posts_index_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'User post index page', type: :feature do
+  let!(:user) { User.create(name: 'Sarkodie', photo: '/path/to/photo1.jpg', post_counter: 10) }
+
+  before do
+    5.times do
+      Post.create(
+        title: 'Post title',
+        text: 'Post text',
+        author_id: user.id,
+        comments_counter: 3,
+        likes_counter: 7
+      )
+    end
+    visit user_posts_path(user)
+  end
+
+  it 'displays user profile information' do
+    expect(page).to have_css("img[src='#{user.photo}'][alt='user photo']")
+    expect(page).to have_content(user.name)
+    expect(page).to have_content("Number of posts: #{user.post_counter}")
+  end
+
+  it 'displays each post with relevant information' do
+    @posts = user.posts
+
+    @posts.each do |post|
+      expect(page).to have_content(post.title)
+      expect(page).to have_content(post.text[0..49])
+      expect(page).to have_content("Comments: #{post.comments_counter}")
+      expect(page).to have_content("Likes: #{post.likes_counter}")
+      expect(page).to have_link('Read post', href: user_post_path(post))
+    end
+  end
```

Comment:
- [ ] This test only confirms that there is a link that can take the User to the `Posts#Show` page when the **_Read Post_** is clicked. Kindly rewrite this test and ensure that you are checking when the User is redirected to the `Posts#Show` page when the **_Read Post_** link is clicked. You can read this Rspec [cheatsheet](https://gist.github.com/zhengjia/428105) to have an idea. 👍 😎 

#### Hint: You can use the `click_link` helper and then confirm the current page is the `Posts#Show` page. 💻 🖱️